velox/docs/functions/spark/datetime.rst

```diff
@@ -7,6 +7,20 @@ Convenience Extraction Functions
 
 These functions support TIMESTAMP and DATE input types.
 
+.. spark:function:: add_months(startDate, numMonths) -> date
+
+    Returns the date that is ``numMonths`` after ``startDate``.
+    Adjusts result to a valid one, considering some months don't have 29th, 30th or 31th
+    day. For example, returns '2015-02-28' adjusted for add_months('2015-01-30', 1) instead
```

Comment:
nit: For example, add_months('2015-01-30', 1) returns '2015-02-28', because 28-th is the last day in February of 2015.